fineract-core/src/main/java/org/apache/fineract/infrastructure/security/utils/SQLInjectionValidator.java

```diff
@@ -180,4 +143,17 @@ private static void patternMatchSqlInjection(String sqlSearch, String lowerCaseS
         }
     }
 
+    private static void validateSQLCommand(String lowerCaseSQL, String[] commands, SQLCommandCondition SQLCondition) {
```

Comment:
please rename `SQLCondition` to match with variable naming conventions. (ex.: conditions)

Reply:
@adamsaghy Thanks for review. I modified the name. Please check 🙇